.eslintrc.js

```diff
@@ -37,6 +37,7 @@ module.exports = {
       files: ['e2e/**/*'],
       rules: {
         'testing-library/prefer-screen-queries': 'off',
+        'testing-library/no-await-sync-query': 'off',
```

Comment:
Why is this eslint rule disabled?

Reply:
Because it's applying the wrong rules in playwright tests. Playwright has its own implementation of "RTL"-like selectors and so certain RTL rules are wrong e.g. in playwright all locators are async so we correctly await them, but this eslint rule is detecting that in RTL `getBy*` are sync functions and so we shouldn't await, despite that these are actually playwright locators (see https://playwright.dev/docs/locators)

We have to do similar for some cypress rules as well as you can see for similar reasons

Reply:
Is it worth using the "override" settings in eslintrc to turn off the rule only for the e2e folder? Alternatively, an `.eslintrc` can be placed in the e2e folder that disables the offending rule only for that folder.

Reply:
This is in the overrides section? And is just turning it off for files in the e2e folder?

Reply:
Ahh yeah, you are right. How did I miss that? Sorry about that.